cloudflare.go

```diff
@@ -35,6 +35,20 @@ func NewZone() *Zone {
 	return &Zone{}
 }
 
+// ZoneIDByName retrieves a zone's ID from the name.
+func (api *API) ZoneIDByName(zoneName string) (string, error) {
+	res, err := api.ListZones(zoneName)
+	if err != nil {
+		return "", err
+	}
+	for _, zone := range res {
+		if zone.Name == zoneName {
```

Comment:
Might be worth checking `zone.Status` too. I'm not sure what would happen if the API user was part of a MU org and the zone existed in multiple accounts/orgs (e.g. if moving a zone between accounts).